project.py

```diff
@@ -336,29 +342,63 @@ def area_code_prefixes() -> list:
     return output_list
 
 
+def show_summary_data(title, data):
+    data_table = {title: data}
+    df = pd.DataFrame(data_table).sort_values(by=title, ascending=False)
+    print(f'=== {title} ===')
+    # If there are more than 10 values show only the top-10
+    print(df.head(10))
+    print('Total', end=': ')
+    print(df[title].sum())
+
+    # make a bar chart of the data
+    # if the dataframe has more than 10 rows, only show the top-10
+    if len(df) > 10:
+        fig_title = f'{title} - Top-10'
+    else:
+        fig_title = title
+    fig = df.head(10).plot.pie(y=title, figsize=(10, 5), title=fig_title, legend=False)
+    # move the legend to the lower left
+    # fig.legend(loc='lower left')
+    # turn off the y-axis label
+    fig.set_ylabel('')
+    # save the chart to a file
+    fig.get_figure().savefig(f'{title}.png')
+    print(f'Chart saved to {title}.png', end='\n\n')
+
+
 if __name__ == '__main__':
+    """Compute summary statistics for the data incident"""
+    # show Aggie pride
     print(show_aggie_pride(), end='\n\n')
 
-    print(area_code_prefixes())
-
     # read the data file
     csv_data = read_csv_file('data.csv')
 
-    # Summarize the credit card types
-    print('Credit Card Types')
-    print(get_credit_card_type(','.join(csv_data['Credit Card'])))
-    print()
+    # Show information about how many credit cards and credit card types were impacted
+    show_summary_data('Credit Card Types', get_credit_card_type(','.join(csv_data['Credit Card'])))
+
+    # Show information about how many email addresses and domains were impacted

```

Comment:
Adding the function 'show_summary_data' was a good way to consolidate all the necessary information that needed to be displayed without having too many lines of code.